src/tesla_ver/bubble_chart/bubble_chart_layout.py

```diff
@@ -76,40 +85,23 @@
                                     id="x_dropdown",
                                     options=[],
                                     value=None,
-                                    placeholder="X Axis Values",
-                                    className="dropdowns",
-                                ),
-                            ]
-                        ),
-                        html.Div(
-                            [
-                                html.P("Sizing Values"),
-                                dcc.Dropdown(
-                                    id="size_dropdown",
-                                    options=[],
-                                    value=None,
-                                    placeholder="Sizing Values",
-                                    className="dropdowns",
-                                ),
-                            ]
-                        ),
-                        html.Div(
-                            [
-                                html.P("Annotation"),
-                                dcc.Dropdown(
-                                    id="annotation_dropdown",
-                                    options=[],
-                                    value=None,
-                                    placeholder="Annotation",
-                                    multi=True,
+                                    placeholder="Choose an Taxa for the X Axis",
```

Comment:
Taxa doesn't need to be capitalized:
"choose a taxa ...."

Reply:
addressed in b5d1e7f